test/setup.js

```diff
@@ -14,28 +14,35 @@
 'use strict';
 
 require('colors');
+var chai = require('chai');
+var sinon = require('sinon');
+var sinonChai = require('sinon-chai');
 var supertest = require('supertest');
 
+process.env.NODE_ENV = 'test';
+
+chai.use(sinonChai);
 // Root object
```

Comment:
I'm not sure this comment is still helpful since the name is more explicit. What do you think?

Reply:
yeah, I think that make sense, i will remove the comment, since the name is more explicit.